cmd/src/users_verify.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"text/template"
+)
+
+const SubCommand = "verify-emails"
```

Comment:
It is better to hard-code this value inline.

Also, having the name be ambiguous like `SubCommand` means it would conflict with every other `cmd/src/*.go` file. And capitalized `SubCommand` means this is exported which is odd